google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/TableResult.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Git thinks this is a new file. Should probably use a new header for 2017/2018 with Google LLC.

Reply:
Done